automated-testing/shadow-testing/readme.md

```diff
@@ -0,0 +1,38 @@
+# Shadow Testing
+
+Shadow testing is one of the useful approaches to reduce risks when you consider to replace the current environment (V-Current) with candidate environment with new feature (V-Next). This approach is monitoring and capturing differences between two environments then compare and reduces all risks before you introduce a new feature/release.
```

Comment:
nit: can you split long text lines into multiple consecutive ones, the MD viewer would render them all as part of the same paragraph either way, but having multiple lines would make it more readable and easier to review during a PR. This applies to other paragraphs as well.

Reply:
Thanks @alkofahi ,fixed this issue.